### ISSUE TITLE
Files missing

### ISSUE THREAD

**Issue by pepa65** [aveferrum/wine-starcraft-installer#5]
It seems the 3 files are no longer at:
```
https://www.demirkan.info/files/sc/wine-staging.tar.xz
https://www.demirkan.info/files/sc/starcraft-icon.png
https://www.demirkan.info/files/sc/winecfg-icon.png
```
Without them the repo is useless. But I still have issues installing StarCraft on my Ubuntu 16.04...

**Comment by edemirkan** [comment]
Script updated to use distro's wine-staging build instead of the custom one. If you're still interested in this, give it a try one more time.

**Comment by pepa65** [comment]
OK, trying this again now on Ubuntu 18.04, following the steps in the readme:
1. No problem
2. The link to the Starcraft installer must have `os=win`, otherwise it doesn't link to the binary, so the whole link should be: `https://battle.net/download/getInstallerForGame?os=win&version=LIVE&gameProgram=STARCRAFT`
3. Also did `sudo dpkg --add-architecture i386` here
4. Had to install `libstxt6:i386`
5. OK, did it twice, because the Wine Gecko installer came up again, and when I clicked Install the script stopped at some point. Second time it went.
6. Didn't work, because I had to install on a different partition (not enough space).
7. No problem, because icons were installed on the Desktop and in the Wine category.

All in all, it works now! Thank you for your work.
